examples/hooks/FieldHooks/ComputeFieldHooks/dateFormat.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { TextField, Sheet, Workbook } from '@flatfile/configure'
+import { SheetTester } from '../../../../src/utils/testing/SheetTester'
+import { format } from 'date-fns'
+
+//This is an example of storing the data hook outside your sheet as a helper function that can be referenced by multiple fields in your sheet.
+//Here we define the function
+const formatDate =
+  (update: string) =>
+  (value: string): string => {
+    try {
+      return format(new Date(value), update)
+    } catch (err) {
+      return value
+    }
+  }
+
```

Comment:
I don't think this code actually runs.  it looks like a function that returns a function.  Are you sure this is what you wanted?

Reply:
oh, I see, that's waht you meant to do.  Well done, you should mention this.

Reply:
I'll be honest, this is code from @hansjhoffman that I rewrote for this example so I don't know how to best explain it 😆

Reply:
This is also an example that I might remove with the launch of SmartDateField, since that would cover the basically same formatting changes this is capable of handling.